styles/index.css

```diff
@@ -0,0 +1,15 @@
+.header__title {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 27px;
+    margin: 25px 25px 0 0;
+}
+
+body {
+    display: grid;
+    font-family: sans-serif;
+    font-size: 18px;
+    margin: 2rem;
+    background:tan;
+}
```

Comment:
Nice work with this function! I noticed you didn't add any styling for grid display or custom classes or ids.